components/form/form.js

```diff
@@ -129,6 +136,9 @@ class Form extends LocalizeCoreElement(LitElement) {
 		if (changedProperties.has('_errors')) {
 			this._hasErrors = this._errors.size > 0;
 		}
+		if ((changedProperties.has('summary-id') || changedProperties.has('_errors')) && this.summaryId && this._hasErrors) {
```

Comment:
I think when `_hasErrors` goes from `true` to `false` we'd still want to set `errors` on the summary so that it clears itself?